Jellyfin.Api/Controllers/ItemsController.cs

```diff
@@ -678,6 +682,7 @@ public ActionResult<QueryResult<BaseItemDto>> GetItemsByUserIdLegacy(
         [FromQuery, ModelBinder(typeof(PipeDelimitedCollectionModelBinder))] string[] genres,
         [FromQuery, ModelBinder(typeof(PipeDelimitedCollectionModelBinder))] string[] officialRatings,
         [FromQuery, ModelBinder(typeof(PipeDelimitedCollectionModelBinder))] string[] tags,
+        [FromQuery, ModelBinder(typeof(PipeDelimitedCollectionModelBinder))] string[] allTags,
```

Comment:
The legacy API operations should not receive new parameters. They are called legacy for a reason